fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(res => res.json())
+      .then(data => this.setState({ users: data }))
+      .catch(err => console.error(err));
+    this.setState({ isLoading: false });
```

Comment:
Still not quite the correct spot. You want to do it after the `fetch` request has completed. So you want to set it in the `.then` callback function.